definitions/fenix.toml

```diff
@@ -585,3 +613,23 @@ GROUP BY 1, 2, 3, 4, 5, 6
 )
 """
 experiments_column_type = "none"
+
+
+[data_sources.feature_usage_metrics_v1]
+from_expression = """(
+    SELECT
+        *
+    FROM `moz-fx-data-shared-prod.fenix.feature_usage_metrics` p
```

Comment:
Should we be using mozdata instead of the `moz-fx-data-shared-prod` GCP project here for views exposed to mozdata??

Reply:
yup, good call